redbot/cogs/economy/economy.py

```diff
@@ -247,6 +247,45 @@ def __init__(self, bot: Red):
                 )
             )
 
+    @_bank.command()
+    @commands.guild_only()
+    @checks.guildowner()
+    async def removedead(self, ctx, confirmation: bool = False):
+        """Delete bank accounts for users no longer in the server"""
+        if await bank.is_global():
+            return await ctx.send(_("This command will not work in a Global bank"))
```

Comment:
Should be changed to something like "This command cannot be used with a global bank." so that it is in the present tense and is more in line with other strings in core.

Also, this is missing a period.

Reply:
Updated string to bring it in line with other strings in core and added period